### PR TITLE
Script Addition : Stargazers

### LINKED ISSUE #973: Track Stargazers, save the info in JSON/CSV.
# Description

So recently our _Project has been trending on GitHub!!!_, Kudos 🥳  

We are about to touch 500 ⭐s as well!
First and foremost, thank you, everyone, it's the cumulative efforts of every contributor to date.

I was really interested in the upward trend and wanted to do some analysis of the recent upsurge. I found a few good projects (see references) for the same.

Those projects are full-fledged codebases consisting of multiple functionalities. I think a simpler script (maybe even a wrapper around GitHub API), that can **list out recent (or all) Stargazers**, **those who unstarred recently**, **Location** (if possible), **email** (we have a good script for that in this repo prepared by @rahulraikwar00) would be a good fit for this project.

## Language

- [x] Go
- [x] Javascript
- [x] Python

## Checklist:

- **Details on the Script**

| name            | about                                                  | labels  | assignees                                                      |
| --------------- | ------------------------------------------------------ | ------- | -------------------------------------------------------------- |
| Track Stargazers | See Description | feature, gssoc21 |  |

**Additional context**: 

Refernces:
- [1](https://github.com/marius92mc/github-stargazers)
- [2](https://github.com/kiok46/Track-Stargazers)
- [3](https://docs.github.com/en/rest/reference/activity)

### PR DESCRIPTION
**Pull request by jhamadhav**
# Description

**Stargazers** gives lists of people that have starred a repo and those who have recently un-starred.

Fixes #973 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation Update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

## Output pic

Output Pic

### CODE REVIEW

**Comment by jhamadhav** [comment]
@vybhav72954 @HarshCasper Please check this PR.
Thank you!

**Comment by jhamadhav** [comment]
@HarshCasper @vybhav72954 I have solved the merge conflict, please review the PR.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "prompt-sync": "^4.2.0"
+  }
+}
```
Comment (HarshCasper) [inline package.json:5]: We don't need to have a `package.json` on the root.

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    "strip-ansi": {
+      "version": "5.2.0",
+      "resolved": "https://registry.npmjs.org/strip-ansi/-/strip-ansi-5.2.0.tgz",
+      "integrity": "sha512-DuRs1gKbBqsMKIZlrffwlug8MHkcnpjs5VPmL1PAh+mA30U0DTotfDZ0d2UUsXpPmPmMMJ6W773MaA3J+lbiWA==",
+      "requires": {
+        "ansi-regex": "^4.1.0"
+      }
+    }
+  }
+}
```
Comment (HarshCasper) [inline package-lock.json:62]: We don't need to have a `package-lock.json` on the root.

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 2 above)
Comment (jhamadhav) [reply package-lock.json:62]: Will remove it asap

**Comment by jhamadhav** [comment]
@HarshCasper I have made the requested changes please check!

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**